system_tests/vision.py

```diff
@@ -190,6 +190,7 @@ def _assert_face(self, face):
 
     def test_detect_faces_content(self):
         client = Config.CLIENT
+        client._use_gax = False
```

Comment:
I realize **why** you don't it, I'm saying don't do it in this way. Just skip the test if you require gRPC or HTTP. See an [example][1]. Note that the skip can't be at import time since it depends on `Config.CLIENT`, which is set **at runtime**, so using `@unittest.skipIf` is a no-go.

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/e7b1414a0b3d1c3d515de1117a18d38d08d55c09/system_tests/bigtable.py#L313-L318